graylog2-web-interface/src/views/components/AdaptableQueryTabs.tsx

```diff
@@ -219,42 +229,55 @@ const AdaptableQueryTabs = ({ maxWidth, queries, titles, selectedQueryId, onRemo
           {tabTitle}
         </NavItem>
       ));
+
+      queriesList.push({
+        id,
+        title,
+      });
     });
 
-    return { navItems, menuItems, lockedItems };
+    return { navItems, menuItems, lockedItems, queriesList };
   }, [lockedTab, onRemove, onSelect, queries, queryTitleEditModal, selectedQueryId, titles]);
 
   useEffect(() => {
     adjustTabsVisibility(maxWidth, lockedTab, setLockedTab);
   }, [maxWidth, lockedTab, selectedQueryId]);
 
   return (
-    <StyledQueryNav bsStyle="tabs" activeKey={selectedQueryId} id="dashboard-tabs">
-      {currentTabs.navItems}
-
-      <NavDropdown eventKey="more"
-                   title={<Icon name="ellipsis-h" />}
-                   className="query-tabs-more"
-                   id="query-tabs-more"
-                   aria-label="More Dashboard Tabs"
-                   noCaret
-                   pullRight
-                   active={openedMore}
-                   open={openedMore}
-                   onToggle={(isOpened) => setOpenedMore(isOpened)}>
-        {currentTabs.menuItems}
-      </NavDropdown>
-
-      {currentTabs.lockedItems}
-
-      <NavItem key="new"
-               eventKey="new"
-               title="Create New Tab"
-               onClick={() => onSelect('new')}
-               className="query-tabs-new">
-        <Icon name="plus" />
-      </NavItem>
-    </StyledQueryNav>
+    <Container>
+      <StyledQueryNav bsStyle="tabs" activeKey={selectedQueryId} id="dashboard-tabs">
+        {currentTabs.navItems}
+
+        <NavDropdown eventKey="more"
+                     title={<Icon name="ellipsis-h" />}
+                     className="query-tabs-more"
+                     id="query-tabs-more"
+                     aria-label="More Dashboard Tabs"
+                     noCaret
+                     pullRight
+                     active={openedMore}
+                     open={openedMore}
+                     onToggle={(isOpened) => setOpenedMore(isOpened)}>
+          {currentTabs.menuItems}
+        </NavDropdown>
+
+        {currentTabs.lockedItems}
+
+        <NavItem key="new"
+                 eventKey="new"
+                 title="Create New Tab"
+                 onClick={() => onSelect('new')}
+                 className="query-tabs-new">
+          <Icon name="plus" />
+        </NavItem>
+      </StyledQueryNav>
+      <IconButton name="cog" onClick={() => setShowConfigurationModal(true)} />
```

Comment:
I'm using IconButton to separate it from Nav because it's not the part of navigation. Also, I placed it on the right side, in my opinion, it looks better there if we don't have a lot of Nav Items. 
I can just add a title to `IconButton`, I think  it would be better